### PR TITLE
feat(ui): Support batch deprecation from the UI (Batch actions part 6/7)

### PR DESCRIPTION
**Pull request by jjoyce0510**
**Summary**
In this PR, we add support for batch deprecating / undeprecating via the UI. 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/entity/shared/EntityDropdown/EntityDropdown.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
 import { Redirect } from 'react-router';
 import { EntityType, PlatformPrivileges } from '../../../../types.generated';
 import CreateGlossaryEntityModal from './CreateGlossaryEntityModal';
-import { AddDeprecationDetailsModal } from './AddDeprecationDetailsModal';
+import { UpdateDeprecationModal } from './UpdateDeprecationModal';
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/entity/shared/EntityDropdown/EntityDropdown.tsx:16]: nice

File: datahub-web-react/src/app/entity/shared/components/styled/search/action/DeprecationDropdown.tsx [hunk 2]
```diff
@@ -1,27 +1,80 @@
...
+                    resources: [...urns.map((urn) => ({ resourceUrn: urn }))],
+                    deprecated: false,
                 },
-            ]}
-            disabled={disabled}
-        />
+            },
+        })
+            .then(({ errors }) => {
+                if (!errors) {
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/entity/shared/components/styled/search/action/DeprecationDropdown.tsx:28]: nit: I think if there are errors then it should skip this block and go straight to the `catch` block?

**Review by chriscollins3456 — APPROVED** [review]
Looks good to me!

File: datahub-web-react/src/app/entity/shared/components/styled/search/action/DeprecationDropdown.tsx (hunk 2 above)
Comment (jjoyce0510) [reply datahub-web-react/src/app/entity/shared/components/styled/search/action/DeprecationDropdown.tsx:28]: Yeah tbh im quite confused abt this

**Pull request merged**